Lesson20/Kadir/app.js

```diff
@@ -9,25 +9,51 @@ form.addEventListener('submit', (event) => {
   // Create a new task item and add it to the list
 
   const listItemElement=document.createElement("li");
-  listItemElement.innerText = input.value;
+  const spanElement = document.createElement("span");
+  spanElement.innerText = input.value;
+  listItemElement.appendChild(spanElement);
   
   // Remove button
   const removeButton = document.createElement("button");
   removeButton.innerText = 'X';
   // TODO: Update styles of removeButton! (use button--light class)
-  listItemElement.appendChild(removeButton);
-
+  
   list.appendChild(listItemElement);
 
   // Add a listener to the task item to toggle its completed status
-  listItemElement.addEventListener('click',() => {
+  spanElement.addEventListener('click',() => {
     // TODO: "completed" class is applied both to the text of the item and the button. Refactor the content of li element so that the class is only applied to the text but not to the button!
-    listItemElement.classList.toggle("completed");
+
+   
+    // listItemElement.classList.toggle("completed");
```

Comment:
In real life, you should not be allowed to leave comments like this in the code, please remove all the code that is not used. You should always be able to find valuable things in the Git history when looking for the commits made in this PR, for example